syntax = "proto2";

message c2s_update_name_response {
	required string name = 1;
}